build.gradle.kts

```diff
@@ -153,6 +153,18 @@ allprojects {
   plugins.withId("org.jetbrains.kotlin.multiplatform") {
     configure<KotlinMultiplatformExtension> {
       jvmToolchain(11)
+      @Suppress("OPT_IN_USAGE")
+      compilerOptions {
+        freeCompilerArgs.addAll("-opt-in=app.cash.zipline.EngineApi")
+      }
+      // https://youtrack.jetbrains.com/issue/KT-61573
+      targets.configureEach {
+        compilations.configureEach {
+          compilerOptions.configure {
```

Comment:
Surprised you need this last `configure` block. I thought `compilerOptions` returned an object directly on which you could invoke `freeCompilerArgs`.